### PR TITLE
DROID-2836 All content | epic

### PR DESCRIPTION
**Pull request by konstantiniiv**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: app/src/main/java/com/anytypeio/anytype/di/feature/AllContentDI.kt [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    fun inject(fragment: AllContentFragment)
+}
+
+@Module
+object AllContentModule {
+
+    @JvmStatic
+    @Provides
+    @PerScreen
+    @Named("AllContent")
```
Comment (uburoiubu) [inline app/src/main/java/com/anytypeio/anytype/di/feature/AllContentDI.kt:57]: What is the reason for this?

File: feature-all-content/src/main/java/com/anytypeio/anytype/feature_allcontent/models/AllContentModels.kt [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+import com.anytypeio.anytype.core_models.primitives.SpaceId
+import com.anytypeio.anytype.domain.misc.UrlBuilder
+import com.anytypeio.anytype.feature_allcontent.presentation.AllContentViewModel.Companion.DEFAULT_INITIAL_SORT
+import com.anytypeio.anytype.presentation.objects.ObjectIcon
+import com.anytypeio.anytype.presentation.objects.getProperName
+import com.anytypeio.anytype.presentation.objects.getProperType
+
+//region STATE
+sealed class AllContentState {
+    data object Initial : AllContentState()
```
Comment (uburoiubu) [inline feature-all-content/src/main/java/com/anytypeio/anytype/feature_allcontent/models/AllContentModels.kt:22]: Init

**Review by uburoiubu — APPROVED** [review]

File: app/src/main/java/com/anytypeio/anytype/di/feature/AllContentDI.kt (hunk 1 above)
Comment (konstantiniiv) [reply app/src/main/java/com/anytypeio/anytype/di/feature/AllContentDI.kt]: deleted

File: feature-all-content/src/main/java/com/anytypeio/anytype/feature_allcontent/models/AllContentModels.kt (hunk 2 above)
Comment (konstantiniiv) [reply feature-all-content/src/main/java/com/anytypeio/anytype/feature_allcontent/models/AllContentModels.kt]: done

**Pull request merged**